superset-frontend/package.json

```diff
@@ -138,12 +139,18 @@
     "d3-scale": "^2.1.2",
     "dom-to-image-more": "^2.10.1",
     "dom-to-pdf": "^0.3.2",
+    "echarts": "^5.4.1",
     "emotion-rgba": "0.0.9",
     "fast-glob": "^3.2.7",
     "fontsource-fira-code": "^4.0.0",
     "fs-extra": "^10.0.0",
     "fuse.js": "^6.4.6",
     "geolib": "^2.0.24",
+    "geostyler": "^12.0.2",
+    "geostyler-data": "^1.0.0",
+    "geostyler-openlayers-parser": "^4.2.1",
+    "geostyler-style": "^7.4.0",
+    "geostyler-wfs-parser": "^2.0.0",
```

Comment:
Please add these libraries to [your plugin directory](https://github.com/apache/superset/pull/25869/files#diff-2a826b513161e937d6993e06b6525786055cf86cc4e43408b7170a5c58a3beef), not to the root directory.

Reply:
I added these to the root directory as they are needed by the `<LayerConfigsControl />` which was added to `superset-frontend/src/explore` and not to the plugin itself. 

The only reason the control is not included in the plugin directory is, that the `<ControlFormItem />` was moved from `@superset-ui/core` to `superset-frontend/src/explore`, so I cannot import it in the plugin directory. This component, however, is needed for a consistent UI of the controls.